io.github.zen_browser.zen.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.zen_browser.zen
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '23.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - mv zen /app/
+
+      - install -Dm0755 metadata/launch-script.sh ${FLATPAK_DEST}/bin/launch-script.sh
+      - install -Dm0644 metadata/policies.json ${FLATPAK_DEST}/bin/distribution/policies.json
+      - install -Dm0644 metadata/icons/io.github.zen_browser.zen.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm0644 metadata/io.github.zen_browser.zen.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm0644 metadata/io.github.zen_browser.zen.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+
+    sources:
+      - type: archive
+        url: https://github.com/zen-browser/desktop/releases/download/1.0.0-a.7/zen.linux.tar.bz2
+        sha256: a0a8877fbdc77285b52a56439289569af09d1bdf323182cfe7c5ec4aac21b339
+        strip-components: 0
+
+      - type: archive
+        url: https://github.com/zen-browser/flatpak/releases/download/1.0.0-a.7/archive.tar
+        sha256: 88a20fac8202848157fc4009085e377301ea941d82ca3829351a760f3879efc7
```

Comment:
```suggestion
        sha256: 88a20fac8202848157fc4009085e377301ea941d82ca3829351a760f3879efc7
        strip-components: 0
```

Reply:
also why isn't the archive not compressed?